gef-ui/server/nunjucks-configuration/index.js

```diff
@@ -1,18 +1,19 @@
+const path = require('path');
 const nunjucks = require('nunjucks');
+let mojFilters = require('@ministryofjustice/frontend/moj/filters/all')();
 const filterLocaliseTimestamp = require('./filter-localiseTimestamp');
 const dashIfEmpty = require('./filter-dashIfEmpty');
 const displayName = require('./filter-displayName');
 const formatAsCurrency = require('./formatAsCurrency');
 const countriesWithEmptyInitialOption = require('./filter-countriesWithEmptyInitialOption');
 const replaceWhiteSpaceWithDash = require('./filter-replaceWhiteSpaceWithDash');
-let mojFilters = require('../../node_modules/@ministryofjustice/frontend/moj/filters/all')();
 
 const configureNunjucks = (opts) => {
   const appViews = [
-    'node_modules/govuk-frontend',
-    'node_modules/@ministryofjustice/frontend',
-    'node_modules/@ministryofjustice/frontend/filters/all',
-    'templates',
+    path.resolve(__dirname, '../../../node_modules/govuk-frontend'),
+    path.resolve(__dirname, '../../../node_modules/@ministryofjustice/frontend'),
+    path.resolve(__dirname, '../../../node_modules/@ministryofjustice/frontend/filters/all'),
+    path.resolve(__dirname, '../../templates'),
```

Comment:
This was needed due to the `govuk-frontend` and `@ministryofjustice` now being hoisted to the root `node_modules`, and also because of the different contexts in which the docker env and component tests would run.

The docker env would now need:
```
'node_modules/govuk-frontend',
'node_modules/@ministryofjustice/frontend',
'node_modules/@ministryofjustice/frontend/filters/all',
'gef-ui/templates',
```
as its context is the root of the project, but the component tests would need:
```
'../node_modules/govuk-frontend',
'../node_modules/@ministryofjustice/frontend',
'../node_modules/@ministryofjustice/frontend/filters/all',
'templates', 
```
as its context is the root of the `gef-ui` directory.

I played around with changing the jest.config to change the root to the root of the project, but that threw up other issues to do with other packages.

By instead basing the path on `__dirname` (i.e. the directory of this file), the the relative path will always be the same, regardless of the run context.